unlock-app/src/stories/creator/CreatorLockForm.stories.js

```diff
@@ -22,17 +22,13 @@ storiesOf('CreatorLockForm', module)
     // TODO: implement this
     const lock = {
       keyPrice: '10000000000000000000',
-      expirationDuration: '172800',
+      expirationDuration: 172800,
       maxNumberOfKeys: 240,
       outstandingKeys: 3,
       address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
       name: 'Existing Lock',
     }
     return (
-      <CreatorLockForm
-        lock={lock}
-        hideAction={() => {}}
-        createLock={() => {}}
-      />
+      <CreatorLockForm hideAction={() => {}} createLock={() => {}} {...lock} />
```

Comment:
I am not sure why you change the `lock={lock}` into `{...lock}`? I think it is probably cleaner to keep the object "as-is" rather than pass its value in order to differentiate between the actual values and the other props which are not lock properties, such as `hideAction` or `createLock`.

Reply:
this has been part of the API for CreatorLockForm since #1029 

If you'd like, it would be relatively trivial to change this post-this merge in another PR, I'm happy to open an issue and self-assign.

Reply:
Ha indeed, it would be much cleaned to use the `lock` object. Please open an issue and asisgn it to yourself and work on that after all is done on this front.

Reply:
#1075